Source/SharpNav/Crowds/Crowd.cs

```diff
@@ -265,73 +263,73 @@ public void Update(float dt)
-					agents[i].TargetState == TargetState.Velocity)
+				if (activeAgents[i].TargetState == TargetState.None ||
+					activeAgents[i].TargetState == TargetState.Velocity)
 					continue;
 
 				//find corners for steering
-				agents[i].Corridor.FindCorners(agents[i].Corners, navQuery);
+				activeAgents[i].Corridor.FindCorners(activeAgents[i].Corners, navQuery);
 
 				//check to see if the corner after the next corner is directly visible 
-				if (((agents[i].Parameters.UpdateFlags & UpdateFlags.OptimizeVis) != 0) && agents[i].Corners.Count > 0)
+				if (((activeAgents[i].Parameters.UpdateFlags & UpdateFlags.OptimizeVis) != 0) && activeAgents[i].Corners.Count > 0)
 				{
-					Vector3 target = agents[i].Corners[Math.Min(1, agents[i].Corners.Count - 1)].Point.Position;
-					agents[i].Corridor.OptimizePathVisibility(target, agents[i].Parameters.PathOptimizationRange, navQuery);
+                    Vector3 target = activeAgents[i].Corners[Math.Min(1, activeAgents[i].Corners.Count - 1)].Point.Position;
+					activeAgents[i].Corridor.OptimizePathVisibility(target, activeAgents[i].Parameters.PathOptimizationRange, navQuery);
 				}
 			}
 
 			//trigger off-mesh connections (depends on corners)
 			for (int i = 0; i < numAgents; i++)
 			{
-				if (agents[i].State != AgentState.Walking)
+				if (activeAgents[i].State != AgentState.Walking)
 					continue;
-				if (agents[i].TargetState == TargetState.None ||
-					agents[i].TargetState == TargetState.Velocity)
+				if (activeAgents[i].TargetState == TargetState.None ||
+					activeAgents[i].TargetState == TargetState.Velocity)
 					continue;
 
 				//check
-				float triggerRadius = agents[i].Parameters.Radius * 2.25f;
-				if (OverOffmeshConnection(agents[i], triggerRadius))
+				float triggerRadius = activeAgents[i].Parameters.Radius * 2.25f;
+				if (OverOffmeshConnection(activeAgents[i], triggerRadius))
 				{
 					//prepare to off-mesh connection
 					int idx = i;
 					
 					//adjust the path over the off-mesh connection
 					NavPolyId[] refs = new NavPolyId[2];
-					if (agents[i].Corridor.MoveOverOffmeshConnection(agents[i].Corners[agents[i].Corners.Count - 1].Point.Polygon, refs, ref agentAnims[idx].StartPos, ref agentAnims[idx].EndPos, navQuery))
+					if (activeAgents[i].Corridor.MoveOverOffmeshConnection(activeAgents[i].Corners[activeAgents[i].Corners.Count - 1].Point.Polygon, refs, ref agentAnims[idx].StartPos, ref agentAnims[idx].EndPos, navQuery))
 					{
-						agentAnims[idx].InitPos = agents[i].Position;
-						agentAnims[idx].PolyRef = refs[1];
-						agentAnims[idx].Active = true;
-						agentAnims[idx].T = 0.0f;
-						agentAnims[idx].TMax = (Vector3Extensions.Distance2D(agentAnims[idx].StartPos, agentAnims[idx].EndPos)
-							/ agents[i].Parameters.MaxSpeed) * 0.5f;
-
-						agents[i].State = AgentState.Offmesh;
-						agents[i].Corners.Clear();
-						agents[i].NeighborCount = 0;
+                        agentAnims[idx].InitPos = activeAgents[i].Position;
```

Comment:
Can you revert these whitespace changes? Should be consistent with the rest of the file